lib/galaxy/webapps/galaxy/api/remote_files.py

```diff
@@ -132,20 +132,18 @@ def __create_jstree( self, directory, disable='folders' ):
         jstree_paths = []
         if os.path.exists( directory ):
             for ( dirpath, dirnames, filenames ) in os.walk( directory ):
-
                 for dirname in dirnames:
                     dir_path = os.path.relpath( os.path.join( dirpath, dirname ), directory )
-                    dir_path_hash = hashlib.sha1( dir_path.encode('utf-8') ).hexdigest()
+                    dir_path_hash = hashlib.sha1(dir_path).hexdigest()
                     disabled = True if disable == 'folders' else False
                     jstree_paths.append( jstree.Path( dir_path, dir_path_hash, { 'type': 'folder', 'state': { 'disabled': disabled }, 'li_attr': { 'full_path': dir_path } } ) )
 
                 for filename in filenames:
                     file_path = os.path.relpath( os.path.join( dirpath, filename ), directory )
-                    file_path_hash = hashlib.sha1( file_path.encode('utf-8') ).hexdigest()
+                    file_path_hash = hashlib.sha1(file_path).hexdigest()
```

Comment:
@nsoranzo Thanks, I'll track back and find out why that difference exists.  Might be reasonable to patch this here, might be much more reasonable to fix it at the source ;)  Either way, I'll follow up and can make the changes in that patch if it looks like the right way to do it.